internal/validations/clusterlogging/validate_clusterlogging_spec.go

```diff
@@ -26,3 +33,30 @@ func validateClusterLoggingSpec(cl v1.ClusterLogging) error {
 	}
 	return nil
 }
+
+//CL(openshift-logging/instance) only - This is a valid LEGACY use case
+//CL(openshift-logging/instance) & CLF(openshift-logging/instance) - This is a valid LEGACY usecase
+//CL(ANY_NS/OTHER) - This is an invalid use case
+//CLF(ANY_NS/ANY_NAME) - This is a valid mCLF use case
+//CL(ANY_NS/ANY_NAME) && CLF(ANY_NS/ANY_NAME) - This is a valid mCLF use case
+func validateSetup(cl v1.ClusterLogging, k8sClient client.Client) error {
```

Comment:
where is this function tied into the entire validation cycle?

Reply:
added here: https://github.com/openshift/cluster-logging-operator/pull/2394/files#diff-d4e83260483dc0e217046eeaa5c5951c2ee39e2989b1fb09faef6cfd5c452465R20 before spec validation